packages/notifications/src/styled/StyledNotification.ts

```diff
@@ -56,7 +56,8 @@ const colorStyles = (props: IStyledNotificationProps & ThemeProps<DefaultTheme>)
  */
 export const StyledNotification = styled(StyledBase).attrs({
   'data-garden-id': COMPONENT_ID,
-  'data-garden-version': PACKAGE_VERSION
+  'data-garden-version': PACKAGE_VERSION,
+  role: 'alert' as string
```

Comment:
We know we want `role="alert"` for `Alerts`. Not so sure about `Notification` – should probably not interrupt the user (aka `aria-live="polite"`). Consult design (@ginnywood) for direction here. We should apply sensible defaults.

Reply:
> We know we want role="alert"

Sounds good, I will keep the `role="alert"` on `Alert`.

> Consult design (@ginnywood) for direction here.

@ginnywood could the `aria-live="polite"` attribute be used as a reasonable default for the `Notification` component since it doesn't interrupt the user? Would consumers ever use a `Notification` and not want the `aria-live="polite"` attribute? 

Alternatively, this component could be left without default for `aria-live`, and the consumers can be responsible for adding these attributes if desired depending on their use case. In this case, I can leave `Notifications` as is, and not make any modifications.

Reply:
Important to note that it can go the other way around. We apply a sensible default and the consumer is able to knock it out using `ariaLive={null}` or `undefined`. But the `attrs` on a Styled component would need to be structured to allow for that.

Reply:
I think the sensible defaults should be:

`Alert` -> `role="alert"`, implicitly has an implicit `aria-live` value of `assertive`.
`Status` -> `role="status"`, implicitly has an implicit `aria-live` value of `polite`.

With these additions, I don't see a need to add props or allow users to knock the `role` attributes off because these `role` attributes map nicely the purposes of the component. An `Alert` will always be used to communicate important information, while a `Notification` communicates information, but is not as urgent as what is communicated in an `Alert`.

@ginnywood @jzempel @austin94, what are your thoughts?

Resources:
https://www.w3.org/TR/wai-aria-1.1/#alert
https://www.w3.org/TR/wai-aria-1.1/#status

Reply:
> Add the ability to knock-out these roles and add testing to prove it is possible. We can't imagine all cases. Providing reasonable defaults is great, but we need to allow users to take control (role={null}), if necessary.

@jzempel thanks for the comments. I have thought a little bit more about the `role` attribute.

I think the `Alert` and `Notification` does not have to foresee all use cases because these components have specific use cases. The `Alert` component use case is to communicate urgent and important information. The `Notification` component use case is similar to `Alert`, but the information is less critical.

If a consumer uses an `Alert` component and is seeking to knock out the `role="alert"` attribute, then an `Alert` component might not be the best design pattern for their implementation. I want to propose that Garden's `Alert` and `Notification` does not provide a  `prop` to change or remove the `role` attribute.

I think other design systems have opted not to allow consumers to change the `role` attribute for the reasons described above. Specifically, I'm looking at [Evergreen's Alert](https://evergreen.segment.com/components/alert), [Polaris's Banner](https://polaris.shopify.com/components/feedback-indicators/banner), and [Material UI's Alert Banner](https://material-ui.com/components/alert/). Polaris makes some opinions on how to treat different types of alerts (`success`, `info`, vs. `critical`) with `role`. Still, the component will always have a `role` attribute that the consumer cannot modify.

Reply:
😕 add `role={null}` to Evergreen. The "role" attribute goes away.

Reply:
@jzempel gotcha, I think an earlier mention of `ariaLive` got me thinking of a custom React `prop`. 

I pushed a commit and made the proper adjustments so the `role` can be overwritten. I opted not to add additional text in the `Alert` and `Notification` examples about modifying the `role` attribute because there doesn't seem to be a common use case for doing so. The native `role` attribute can now be overwritten.